notebooks/Wczytanie_danych.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
Po co nam drugi plik wczytanie danych? Do czego ma on służyć? Czy to nie niepotrzebne zduplikowanie notatników?

Reply:
Temat pod dyskusje. Idea byłą taka, że możemy mieć też lokalne środowisko do developmentu - oprócz colaba. Ale może chcemy się np. ograniczyć tylko do puszczania testów na takim lokalnym sparku.

Reply:
Tak czy siak chciałbym ten notatnik zostawić jako przykład koniecznej konwersji między podejściami.